accounts/sso/index.mdx

```diff
@@ -23,9 +23,11 @@ integration](/docs/accounts/sso/saml2) steps.
 ## Default membership
 
 New users that authenticate through SSO for the first time will receive the
-standard "View & Modify (All Servers)" permission in your organization. You
-currently cannot change this default, but you can assign new roles after the
-user has signed in successfully for the first time.
+standard "View & Modify (All Servers)" permission in your organization, unless
+you specify the default role during the integration setup. You can also change
```

Comment:
Suggestion for the suggestion: :wink: 

> You can change the default role after completing the integration setup, but note that this will _not_ change the role assigned to users who have already authenticated through SSO.

(Or we can have that as a separate sentence, but I think it's good to mention that.)